JavaScript_Basics/math.js

```diff
@@ -0,0 +1,53 @@
+//Math is a built-in object that has properties and methods for mathematical constants and functions. Not a function object.
+
+
+//Math.round(number) rounds off the value of number to nearest integer.
+//when a string input is provided it says "NaN"
+//It can take number in string format as well.
+let num1 = 24.45;
+let num2 = 24.5;
+let num3 = 24.95;
+let num4 = "5.6";
+let num5 = "hello";
+
+num1 = Math.round(num1);
+console.log(num1);
```

Comment:
@sneha122 Please add the results of such console.log statements so that anyone can understand after seeing the file